src/main/java/com/example/LooseCouplingDemo/controller/MessagesController.java

```diff
@@ -0,0 +1,37 @@
+package com.example.LooseCouplingDemo.controller;
+
+import com.example.LooseCouplingDemo.dto.AddMessagesDTO;
+import com.example.LooseCouplingDemo.service_Inteface.MessageServiceInterface;
+import com.example.LooseCouplingDemo.service_implementation.MessageService;
+import jakarta.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/messages")
+public class MessagesController {
+    @Autowired
+    @Qualifier("email")
+    MessageServiceInterface messageServiceInterface;
+
+
+//    @Autowired
+//    MessageService messageService;
+    @GetMapping
+    List<AddMessagesDTO> getALlMessages(){
```

Comment:
```suggestion
    List<AddMessagesDTO> getAllMessages(){
```
Follow camelcase naming convention

Reply:
Done